core/variant.h

```diff
@@ -396,6 +396,7 @@ class Variant {
 		OP_NOT,
 		//containment
 		OP_IN,
+		OP_NOT_IN,
```

Comment:
Sure, that works. I thought it would fit well with the other boolean operators (== and !=, > and <= and so on) and it would be a nicely shorter code-path during runtime for the price of adding the operator.

I'll change it to generate two operations.